imports/ui/pages/components/RightSidebar/PlayBlock.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from "react";
+import { Meteor } from "meteor/meteor";
+import { Button } from "antd";
+import PlayFriendOptions from "./PlayFriendOptions";
+import PlayWithFriend from "./PlayWithFriend";
+import PlayChooseBot from "./PlayChooseBot";
+import GameHistory from "./elements/GameHistory";
+import { GameControlBlock } from "./elements/GameControlBlock";
+import { translate } from "../../../HOCs/translate";
+
+class PlayBlock extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      status: "none"
+    };
+  }
+
+  handlePlayWithFriend = () => {
+    this.setState({ status: "play-friend-options" });
+  };
+
+  handlePlayFriendOptions = data => {
+    this.setState({ status: "play-with-friend", options: data });
+  };
+
+  handleChooseFriend = friendId => {
+    const { onChooseFriend } = this.props;
+    const { options } = this.state;
+
+    onChooseFriend({ friendId, options });
+  };
+
+  handlePlayComputer = () => {
+    this.setState({ status: "choose-bot" });
+  };
+
+  hanldePlayWithBot = data => {
+    const { onBotPlay } = this.props;
+    const { ratingType, skillLevel, color, incrementOrDelayType, initial, incrementOrDelay } = data;
+
+    onBotPlay(
```

Comment:
Use object as argument to pass multiple arguments into function. It will improve readability. In this case I don't know what `0` is stands for.

In this case: ```onBotPlat({
  score: 0,
  ratingType,
  initialScore: initial,
 ...
});
```
readability will be improved